web/template/layout/pagination-bottom.gotmpl

```diff
@@ -17,17 +17,22 @@
       <li class="moj-pagination__item moj-pagination__item--prev">
```

Comment:
Can the bottom and top pagination now be put into one gotmpl so you aren't duplicating code?

Reply:
@johnnolan The reason we did this was that it failed pa11y, we have removed pa11y so could in theory put it in for now and leave it as a future problem? But we couldn't think of another solution.

Reply:
Yeah sounds like a good idea, I imagine the pa11y thing was to do with unique naming of ids or similar, but there is an easy way around that. Looking ace!